mmdet3d/core/post_processing/box3d_nms.py

```diff
@@ -133,3 +135,45 @@ def aligned_3d_nms(boxes, scores, classes, thresh):
 
     indices = boxes.new_tensor(pick, dtype=torch.long)
     return indices
+
+
+@numba.jit(nopython=True)
```

Comment:
Use torch rather than numpy to implement `circle_nms` since torch could be faster.